.buildkite/scripts/build.ps1

```diff
@@ -1,3 +1,92 @@
+$stack_version="8.7.0"
+
+echo "~~~ Installing dotnet-sdk"
 & "./tools/dotnet-install.ps1" -NoPath -JSonFile global.json -Architecture "x64" -InstallDir c:/dotnet-sdk
 ${env:PATH} = "c:\dotnet-sdk;" + ${env:PATH}
-./build show
\ No newline at end of file
+Get-Command dotnet | Select-Object -ExpandProperty Definition
+
+echo "~~~ Reading msi certificate from vault"
+$MsiCertificate=& vault read -field=cert secret/ci/elastic-elastic-stack-installers/msi
+$MsiPassword=& vault read -field=password secret/ci/elastic-elastic-stack-installers/msi
+Remove-Item Env:VAULT_TOKEN
+
+$cert_home="C:/.cert"
+New-Item $cert_home -Type Directory -Force
+[IO.File]::WriteAllBytes("$cert_home/msi_certificate.p12", [Convert]::FromBase64String($MsiCertificate))
+[IO.File]::WriteAllText("$cert_home/msi_password.txt", $MsiPassword)
+echo "Certificate successfully written to $cert_home"
+
+$client = new-object System.Net.WebClient
+$currentDir = $(Get-Location).Path
+$beats = @('auditbeat', 'filebeat', 'heartbeat', 'metricbeat', 'packetbeat', 'winlogbeat')
+$ossBeats = $beats | ForEach-Object { $_ + "-oss" }
+$kinds =  @("-SNAPSHOT")
+if ($env:BUILDKITE_BRANCH -match '^\d+\.\d+$') {
```

Comment:
This will not have an effect on PRs and main builds, but for release branches we'll build both staging and snapshot builds in one go.